flexi_config/config.py

```diff
@@ -53,16 +53,33 @@ def get(cls, key):
                     return get_secret(parts_of_key[1])
                 elif len(parts_of_key) == 3:
                     return get_specific_secret(parts_of_key[2], parts_of_key[1])
+            elif value.startswith("cache"):
+                parts_of_key = value.split(":")
+                if len(parts_of_key) == 3:
+                    secret = cls.fetch_secret_value_from_cache(parts_of_key[2])
+                    return secret
+                elif len(parts_of_key) == 4:
+                    secret = cls.fetch_secret_value_from_cache((parts_of_key[2]))
+                    return secret.get(parts_of_key[3])
         elif value is None:
-            key_parts = key.split('.')
+            key_parts = key.split(".")
             for i in range(1, len(key_parts)):
-                k = '.'.join(key_parts[:-i])
+                k = ".".join(key_parts[:-i])
                 v = cls.get(k)
                 if v is not None and i < len(key_parts) - 1:
-                    value = jmespath.search('.'.join(key_parts[i + 1:]), v)
+                    value = jmespath.search(".".join(key_parts[i + 1 :]), v)
                     break
             else:
                 return None
         return value
 
+    @classmethod
```

Comment:
This should be `@staticmethod` because it doesn't make use of the reference to the class